common/beerocks/bwl/dummy/base_wlan_hal_dummy.h

```diff
@@ -75,6 +75,8 @@ class base_wlan_hal_dummy : public virtual base_wlan_hal,
     bool dummy_send_cmd(const std::string &cmd, char **reply); // for external process
     bool dummy_send_cmd(const std::string &cmd);
 
+    static const int predefined_vaps_num = 4;
```

Comment:
Just so I don't forget - we should make sure we are using this value in the M1, and not hard coded in the agent.

Reply:
"We should" as in "we should add a bwl function to report the max number of VAPs", right? Because currently this is indeed hard coded in the agent. -> #810

Reply:
Added to the sprint backlog and to the first milestone.

Reply:
Maybe max VAPs supported by the radio could be useful as a parameter coming from the folders emulating the radios rather than some fixed value. I don't know if there is any value in this to test different limits, just a thought.

Reply:
Good one! Updated #810 with that idea.